### PR TITLE
Allow association of other file formats

### PR DESCRIPTION
**Pull request by lepickle**
I have changed some of the code involving adding an list of known formats, and adding video target folder. I have also changed the conditions after scanning the files so the file extension can be compared to an array instead of individually. I have also renamed some of the folders to better classify them into their own

### CODE REVIEW

**Comment by lepickle** [comment]
(continuation) target folders. All format information were taken from wikipedia. Some of them aren't known commonly so I have excluded them for now.

File: bin/classifier [hunk 1]
```diff
@@ -2,17 +2,36 @@
 import os
 
 directory = os.getcwd()
-target_pdf = directory+'/pdfs'
+# target_pdf = directory+'/pdfs' PDFs are now moved to docs
```
Comment (lepickle) [inline bin/classifier]: PDF files are now organized to 'Documents' folder

**Comment by lepickle** [comment]
I tested and I found some errors but I have already fixed it on e73af25. Tested with fix showed no more errors. Feel free to test some more.

**Comment by bhrigu123** [comment]
Thanks for helping out.
But could you squash these last commits into one single commit.

**Comment by lepickle** [comment]
Welcome :)
Sure. i somehow forgot to just append the commits before pushing them.

**Pull request closed without merging**